roles/deprovision-keycloak-apb/tasks/cleanup-keycloak.yml

```diff
@@ -14,19 +14,20 @@
   shell: "{{ kc_namespace_configmap }} -o jsonpath={.data.public_client_id}"
   register: kc_configmap_user
 
-- name: "Generate KeyCloak auth token"
+- name: Generate KeyCloak auth token
   uri:
     url: "{{ kc_configmap_uri.stdout }}/auth/realms/master/protocol/openid-connect/token"
     method: POST
     body: "client_id=admin-cli&username={{ ADMIN_NAME }}&password={{ ADMIN_PASSWORD }}&grant_type=password"
     validate_certs: no
-  register: keycloak_auth_response
-  until: keycloak_auth_response.status == 200
+  register: kc_auth_response
   retries: 20
   delay: 2
+  until: kc_auth_response.status == 200
+  ignore_errors: yes
```

Comment:
I think we should handle 400s like 401 403

Reply:
Is there a particular status we can check for if the kc service is not running? Seems we could check this easy by reducing the retries and setting up a shared service then deleteing the keycloak

Reply:
it should be 503. But I didnt try that

Reply:
ok This looks better. Only one other suggestion. Should the 503 check be only when USE_SHARED_SERVICE is present?